tests/test_aten_t.py

```diff
@@ -0,0 +1,122 @@
+import torch
+import torch_ttnn
+import unittest
+from torch_ttnn import ttnn
```

Comment:
can we do simply `import ttnn`?

Reply:
Yeah that can be done as well.

Reply:
Just FYI, mock_ttnn will be removed during the refactor very soon.